vite.config.mts

```diff
@@ -77,7 +79,7 @@ export default defineConfig({
     port: 4000,
     proxy: {
       "/api": {
-        target: process.env.CARE_API ?? "https://careapi.ohc.network",
+        target: local_care_api ?? "https://careapi.ohc.network",
```

Comment:
Why was this changed? The default dev. behaviour would become to use local backend with this change right? Could you revert it if it was unintentional?

Reply:
Yeah I can revert it. The reason for this change was related to the previous problem which I mentioned, that the local backend wasn't connecting to the frontend through the env variable.

Reply:
OK, I ran "CARE_API=http://127.0.0.1:9000/ bun run dev " and its working perfectly fine. I will push the requested changes.

Reply:
@rithviknishad

Reply:
Could you please mention the command and the way for resolving the failing cypress tests.

Reply:
See README: https://github.com/coronasafe/care_fe#-run-cypress-tests

```
bun run cypress:open
```

Reply:
Assuming dev. server is already running connected to your local BE

Reply:
Raise a seperate issue in case `CARE_API=... npm/yarn/pnpm dev` is not working.